pkg/util/keyer/dashboard.go

```diff
@@ -45,7 +44,7 @@ func (d Dashboard) PrettyPrint(uncompressed, useHex bool) {
 
 	if d.pubKey != nil {
 		if uncompressed {
```

Comment:
Frankly, this branch doesn't have a lot of reasons to exist. But let's have it for now.